packages/atomic/src/locales.json

```diff
@@ -8289,5 +8289,59 @@
     "zh": "空的搜索",
     "zh-CN": "空的搜索",
     "zh-TW": "空的搜索"
+  },
+  "no-products": {
+    "en": "No products",
+    "fr": "Aucun produit",
+    "cs": "Žádné produkty",
+    "da": "Ingen produkter",
+    "de": "Keine Produkte",
+    "el": "Δεν υπάρχουν προϊόντα",
+    "es": "No hay productos",
+    "fi": "Ei tuotteita",
+    "hu": "Nincs termék",
+    "id": "Tidak ada produk",
+    "it": "Nessun prodotto",
+    "ja": "商品がありません",
+    "ko": "제품 없음",
+    "nl": "Geen producten",
+    "no": "Ingen produkter",
+    "pl": "Brak produktów",
+    "pt": "Sem produtos",
+    "pt-BR": "Sem produtos",
+    "ru": "Нет продуктов",
+    "sv": "Inga produkter",
+    "th": "ไม่มีสินค้า",
+    "tr": "Ürün yok",
```

Comment:
Sounds like Tolkien Mordor language.  I dig it.

Reply:
We're missing `tlh` support :/